index.rst

```diff
@@ -352,28 +385,145 @@ To make sure the system would work properly we had to enable two additional feat
+For instance, DDS allows applications to share substantial resources when creating readers and writers on the same partition.
+At the same time, the software architecture allows us to safely share most of those resources for an individual component so, splitting the topics for a component into more partitions means there would be less shared resources.
+
+Details on how this is implemented can be found in `SalObj documentation <https://ts-salobj.lsst.io>`__.
+
+.. _Quality-of-Service-configuration:
+
+Quality of Service configuration
+--------------------------------
+
+The Quality of Service (QoS) are topic specify configurations that control many aspects of how information is handled by DDS, including how reliable messages are exchanged and if a topic has historical data (for later joiners).
+Our architecture apply QoS configuration based on the message type, e.g., commands, events, telemetry and acknowledgements.
+
+Up to SAL v4 and SalObj v5 the QoS was partially stored in a `QoS file <https://raw.githubusercontent.com/lsst-ts/ts_idl/v1.4.0/qos/DDS_DefaultQoS_All.xml>`__ and partially hard coded into the software code base.
+This created some issues, specially when changes needed to be done on the QoS, which required full rebuild of the system.
+As of SAL v5 and SalObj v6 this settings will be fully loaded from a QoS file stored in the `ts_idl repository <https://github.com/lsst-ts/ts_idl>`__.
+
+As part of this exercise we adopted a couple changes in the previous QoS settings.
+The most important change to be noted is that now Telemetry topics have durability set to ``VOLATILE``, which means the system will no longer store historical data.
+Since telemetry is continuously published at a set frequency by CSCs, storing historical data gives little to no benefit while imposes an additional load on the DDS systems.
+
+From now on, events are the only set of topics that are configured with ``TRANSIENT`` durability, which allows them to retain historical information.
+
+.. _Nublado:
+
+Nublado
+-------
+
+After the initial test it seemed like the Jupyter notebooks from the nublado platform would also have to transition to using the DDS daemon, and therefore, the licensed version of OpenSplice.
+During the tests of the features introduced above it was possible to demonstrate that the single process mode is now capable of joining a domain and communicate with the CSCs in the system.
+In particular, the feature that seems to allow the use of single process mode is the introduction of the :ref:`DDS data partitioning <Implementing-a-new-DDS-data-partitioning-schema>`.
+
+This considerably simplify handling the nublado configuration as the containers can continue to be hosted in public repositories, there is no need to configure them to use shared memory and to worry about users notebooks crashing and corrupting the daemons they are attached to, potentially affecting other systems.
```

Comment:
Did you describe the corruption issue above? If so I missed it. This is crucial information.

In particular we know we must send SIGTERM, not SIGKILL, to applications using shared memory mode.